test/fixtures/cli/nyc-config-js/nycrc-config.js

```diff
@@ -1,9 +1,13 @@
 module.exports = {
-  'check-coverage': true,
-  'per-file': true,
-  lines: 100,
-  statements: 100,
-  functions: 100,
-  branches: 100,
-  exclude: []
+  toNYCRC() {
```

Comment:
@JaKXz any suggestion on if/how this should be documented?

Reply:
I would write something in `docs/` with a link to it at the bottom of the readme.

Reply:
I added the link below the section about publishing reusable configs, this seemed like a better place than at the very end.  Let me know what you think.